frontend/app/Entry/MediaPreview.tsx

```diff
@@ -135,7 +141,7 @@ const MediaPreview:React.FC<MediaPreviewProps> = (props) => {
         </div>
     }
 
-    return <div className={classes.centered}>
```

Comment:
hmm i think i already changed this to lift the class definition to the caller, i.e. this becomes `className={props.className}` and you define `centeredMedia` in the container and pass it in with `<MediaPreview className={classes.centeredMedia} ....>`

This is because `<MediaPreview>` is also used in the EntryView component for a full-sized player so it needs slightly different CSS there.

Reply:
I have fixed this.